pynest/examples/sensitivity_to_perturbation.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+#
+# sensitivity_to_perturbation.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+'''
+Sensitivity to perturbation
+---------------------------
+
+This script simulates a network in two successive trials which
+are identical except for an extra input spike in the second realisation.
+(a small perturbation). The network consists of recurrent, randomly
+connected excitatory and inhibitory neurons. Its activity is driven
+by an external Poisson input provided to all neurons independently.
+In order to ensure that the network is reset appropriately between
+the trials, we do the following steps:
+- resetting the network
+- resetting the random network generator
+- resetting the internal clock
+- deleting all entries in the spike detector
+- introducing a hyperpolarisation phase between the trials
+  (in order to avoid that spikes remaining in the NEST memory
+  after the first simulation are fed into the second simulation)
+'''
+
+'''
+Importing all necessary modules for simulation, analysis and plotting.
+'''
+
+import numpy
+import pylab 
+import nest
+
+'''
+Here we define all parameters necessary for building and simulating
+the network.
+We start with the global network parameters. 
+'''
+
+NE = 1000      # number of excitatory neurons
+NI = 250       # number of inhibitory neurons
+N = NE + NI
+KE = 100       # excitatory in-degree
+KI = 25        # inhibitory in-degree
+
+'''
+Parameters specific for the neurons in the network. The  default values of
+the reset potential "E_L" and the spiking threshold "V_th" are used to set
+the limits of the initial potential of the neurons.
+'''
+
+neuron_model = 'iaf_psc_delta' 
+neuron_params = nest.GetDefaults(neuron_model) 
+Vmin = neuron_params['E_L']   # min. potential for distribution of initial potentials (mV)
+Vmax = neuron_params['V_th']  # max. potential for distribution of initial potentials (mV)
+
+'''
+Synapse parameters. Changing the weights (J) in the network can lead to
+qualitatively different behaviors. If J is small (e.g. J = 0.1) we
+are likely to observe a non-chaotic network behavior (after perturbation
+the network returns to its original activity). Increasing J (e.g J = 3.)
+leads to rather chaotic acitivity.
+'''
+
+J = .1                    # excitatory synaptic weight (mV)
```

Comment:
Given the examples end up on the home page, it would be good to select a value for J that gives 'interesting' results - does 0.1 do this? If not, I suggest changing it.